cms/templates/certificate_page.html

```diff
@@ -73,13 +73,15 @@ <h2>Congratulations, {{ learner_name }}!</h2>
                     {% endif %}
                     {% if page.CEUs %}
                         <span class="award-text">Awarded {{ CEUs }} Continuing Education Units (CEUs) <br>
-                            {% if is_program_certificate %}
-                                {{ end_date|date }}
-                            {% else %}
-                                {{ start_date|date }} - {{ end_date|date }}
-                            {% endif %}
                         </span>
                     {% endif %}
+                    <span>
```

Comment:
I think we need to style this too like before e.g. when we have CEUs set the dates to display styled as:
**(BEFORE)**
<img width="504" alt="Screenshot 2022-09-15 at 3 37 31 PM" src="https://user-images.githubusercontent.com/34372316/190385170-a832dc66-52c6-4fc7-85d2-289cd5d3457b.png">

After this PR, the dates display in small font size:
**AFTER**
<img width="450" alt="Screenshot 2022-09-15 at 3 37 48 PM" src="https://user-images.githubusercontent.com/34372316/190384584-c9fc4ac5-8de7-48d0-98c4-2d7bf7210594.png">

 There are two issues in this, as seen in the above images:
1. The space between CEUs and dates tests has increased
2. The font size/style of the dates text has changed

Reply:
Fixed